docs/source/conceptual_guides/quantization.md

```diff
@@ -0,0 +1,60 @@
+
+  
+# Motivation 
+
+Regardless of the quantization method chosen, our main motivation is to achieve the following:
+
+* Improve the Inference speed :  Training deep neural networks is a computationally intensive task, involving complex mathematical operations across numerous parameters. The 
+computational expense becomes particularly apparent during the inference phase, where the trained model makes predictions on new data. Inference speed is crucial for real-time applications, such as image recognition in mobile apps or voice assistants.
+
+  For instance, imagine a deep learning model responsible for identifying objects in a live video stream. The model needs to process each frame quickly to provide timely predictions. This rapid processing becomes challenging when dealing with large models that have a high number of parameters, such as state-of-the-art language models.
+
+  Here's where quantization comes to the rescue. By reducing the precision of the model's parameters during inference, we can significantly cut down on the computational resources required. Specifically, moving from the standard 32-bit floating-point representation (float32) to 8-bit integer representation (int8) allows us to perform computations with fewer bits, resulting in faster and more efficient inference without sacrificing essential information.
+
+
+* The need for quantized models : There is an increasing demands to run these huge models on our laptops or small devices such as  mobile phone . 
+  
+  None of the above is possible without **Quantization**   
+
+
+## Supported Tuners to perform quantization in 🤗 Hugging face PEFT
+
+🤗 PEFT provides different fine-tuning techniques  to perform quantization:
+
+* The `LoRA` techniques allows us to
+[quantize and fine-tune models ](https://huggingface.co/docs/peft/main/en/package_reference/lora)by reducing the number of trainable parameters.
+
+* The `AdaLora` techniques also allows us to [quantize and fine-tune models](https://huggingface.co/docs/peft/main/en/package_reference/adalora) but unlike LoRA, which distributes parameters evenly across all modules. More parameters are budgeted for important weight matrices and layers while less important ones receive fewer parameters.
+
+* The `ia3` allows us to [quantize and fine-tune models](https://huggingface.co/docs/peft/main/en/package_reference/ia3) it is a method that adds three learned vectors to rescale the keys and values of the self-attention and encoder-decoder attention layers, and the intermediate activation of the position-wise feed-forward network.
+
+
+## References
+
+- [All about Quantization ](https://www.youtube.com/watch?v=UQlsqdwCQdc&t=478s) you-tube video
+- [LLM Series — Parameter Efficient Fine Tuning](https://medium.com/@abonia/llm-series-parameter-efficient-fine-tuning-e9839fae44ac)
```

Comment:
```suggestion
- [LLM Series — Parameter Efficient Fine Tuning](https://medium.com/@abonia/llm-series-parameter-efficient-fine-tuning-e9839fae44ac)
- [🤗 transformers quantization documentation page](https://huggingface.co/docs/transformers/main_classes/quantization) 
```

Reply:
I'll do that as soon as possible

Reply:
I'll do that as soon as possible

Reply:
you can just click into `"Commit Suggestion"` and the suggestion will be added directly